ttn/ttnmqtt.py

```diff
@@ -100,22 +73,18 @@ def connect(self):
         self.__client.username_pw_set(self.__app_id, self.__access_key)
 
         if self.__mqtt_address is None:
-            creds = grpc.ssl_channel_credentials()
             if self.__discovery_address is None:
-                self.__discovery_address = 'discovery.thethings.network:1900'
-            channel = grpc.secure_channel(self.__discovery_address, creds)
-            stub = discovery_pb2_grpc.DiscoveryStub(channel)
-            req = discovery_pb2.GetByAppIDRequest()
-            req.app_id = self.__appID
-            res = stub.GetByAppID(req)
+                self.__discovery_address = "discovery.thethings.network:1900"
```

Comment:
Instead of checking if an attribute is `None`, use [default values](http://www.effbot.org/zone/default-values.htm)